### DISCUSSION TITLE
[BUG] Mirth 4.5.0 is not working properly.

### DISCUSSION THREAD

**Discussion by Vishal224-Agl** [nextgenhealthcare/connect#6286] [Q&A]
**Describe the bug**
Mirth channels are not getting save and not able to deploy channels, when click on save no action is getting performed rest of the buttons are working fine.

**To Reproduce**
Setup steps (if required). Example:
1. Create new channel, make some changes and then click on save channel, or try to deploy channel.

**Expected behavior**
Changes should save and able to deploy channel.

**Actual behavior**
Save and Deploy buttons are not working rest of them are working fine.

**Screenshots**
<img width="1728" alt="image" src="https://github.com/user-attachments/assets/28055213-f3e8-4985-b68e-767f3d8d23e2">

**Comment by pacmano1** [comment]
@Vishal224-Agl , I moved your "issue" to the discussions area.  There are likely hundreds (or thousands) of installs of 4.5 running without a problem.  But on MacOS ARM processors, the mirth admin tool can misbehave quite badly, there are already issues posted on that and no fix yet.   If you are running an M1, M2, M3 mac, please try the admin tool from some other machine not on that architecture.   

Also, assuming you are using the Mirth Admin Launcher to launch the admin tool, check the "Show Java Console" box to see if java is erroring.  There may be other issues, but you are not posting enough details about your environment to get proper help.
